test/dotcom/utils/date_time_test.exs

```diff
@@ -0,0 +1,117 @@
+defmodule Dotcom.Utils.DateTimeTest do
+  use ExUnit.Case
+  use ExUnitProperties
+
+  import Dotcom.Utils.DateTime
+  import ExUnit.CaptureLog
+  import Test.Support.Generators.DateTime
+
+  describe "timezone/0" do
+    test "returns a valid timezone for the application" do
+      # Verify/Verify
```

Comment:
(Non-blocking) Should this be `Exercise/Verify`? Or is there a convention that I missed?